File: Backend/Backend/Models/BusinessCard.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Xml.Serialization;

namespace BusinessCardManagement.Backend.Models
{
	[Serializable]
	[XmlRoot("BusinessCard")] 
	public class BusinessCard
	{
		[XmlElement("Id")] 
		public int Id { get; set; }

		[XmlElement("Name")]
		[Required]
		public string? Name { get; set; }

		[XmlElement("Gender")]
		[Required]
		public string? Gender { get; set; }

		[Required]
		[XmlElement("DOB")] 
		public DateTime DOB { get; set; }

		[Required]
		[XmlElement("Email")] 
		public string? Email{ get; set; }

		[Required]
		[XmlElement("Phone")] 
		public string? Phone { get; set; }

		[Required]
		[XmlElement("Address")] 
		public string? Address { get; set; }

		[XmlElement("Photo")] 
		public byte[]? Photo { get; set; }
	}
}

File: ClientApp/src/app/Models/BusinessCard.ts
export interface BusinessCard {
    id?: number; // Primary Key
    name?: string; // Optional Name of the cardholder
    gender?: "Male | Female"; // Optional Gender of the cardholder
    dob?: Date; // Date of Birth (use Date type for handling dates)
    email?: string; // Optional Email of the cardholder
    phone?: string; // Optional Phone number of the cardholder
    address?: string; // Optional Address of the cardholder
    photo?: string; // Optional Photo of the cardholder (string for base64 representation)
}
